SrcModules/Network/src/Network.cpp

```diff
@@ -10,15 +10,19 @@ extern "C" zia::api::Net * create()
 
 bool nzm::Network::config(const zia::api::Conf &conf)
 {
-  nz::Log::debug("CONFIG NETWORK LOADING");
-  return false;
+  try { this->_port = std::get<long long>(conf.at("port").v); }
+  catch (std::exception&) {
+      nz::Log::warning("port not found or must be a number, default port set to '80'", "Module Network", 100);
```

Comment:
the last param of log::warning is a boolean, not a int

Reply:
And don't catch a std::exception &, it generate warning on windows, use "..." instead